$MirOS$

	When will people learn to use editors like jupp that
	disable word wrap by default?

--- aclocal.m4.orig	Mon Apr  7 13:16:44 2008
+++ aclocal.m4	Mon Apr  7 13:16:53 2008
@@ -601,16 +601,11 @@ int main (int argc, char *argv[])
     }
   else
     {
-      printf("\n*** 'smpeg-config --version' returned %d.%d.%d, but the 
-minimum version\n", $smpeg_major_version, $smpeg_minor_version, 
-$smpeg_micro_version);
-      printf("*** of SMPEG required is %d.%d.%d. If smpeg-config is 
-correct, then it is\n", major, minor, micro);
+      printf("\n*** 'smpeg-config --version' returned %d.%d.%d, but the minimum version\n", $smpeg_major_version, $smpeg_minor_version, $smpeg_micro_version);
+      printf("*** of SMPEG required is %d.%d.%d. If smpeg-config is correct, then it is\n", major, minor, micro);
       printf("*** best to upgrade to the required version.\n");
-      printf("*** If smpeg-config was wrong, set the environment variable 
-SMPEG_CONFIG\n");
-      printf("*** to point to the correct copy of smpeg-config, and remove 
-the file\n");
+      printf("*** If smpeg-config was wrong, set the environment variable SMPEG_CONFIG\n");
+      printf("*** to point to the correct copy of smpeg-config, and remove the file\n");
       printf("*** config.cache before re-running configure\n");
       return 1;
     }
